### ISSUE TITLE
2.2.4 stable release fails to find INT_MAX 

### ISSUE THREAD

**Issue by thefamouseric** [libgd/libgd#366]
./configure; make
.......

depbase=`echo gd_gd2.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -std=gnu99 -DHAVE_CONFIG_H -I.   -I/usr/include/libpng15    -I/usr/include/freetype2    -I/usr/include/freetype2         -g -O2 -fvisibility=hidden -Wall -MT gd_gd2.lo -MD -MP -MF $depbase.Tpo -c -o gd_gd2.lo gd_gd2.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -std=gnu99 -DHAVE_CONFIG_H -I. -I/usr/include/libpng15 -I/usr/include/freetype2 -I/usr/include/freetype2 -g -O2 -fvisibility=hidden -Wall -MT gd_gd2.lo -MD -MP -MF .deps/gd_gd2.Tpo -c gd_gd2.c  -fPIC -DPIC -o .libs/gd_gd2.o
gd_gd2.c: In function ‘_gd2GetHeader’:
gd_gd2.c:212:40: error: ‘INT_MAX’ undeclared (first use in this function)
   if (*ncx <= 0 || *ncy <= 0 || *ncx > INT_MAX / *ncy) {
                                        ^
gd_gd2.c:212:40: note: each undeclared identifier is reported only once for each function it appears in

[root@pbijcyvmlnag01 libgd-2.2.4]# uname -a
Linux pbijcyvmlnag01 3.10.0-514.el7.x86_64 #1 SMP Wed Oct 19 11:24:13 EDT 2016 x86_64 x86_64 x86_64 GNU/Linux

**Comment by cmb69** [comment]
Duplicate of #360.
